sktime/dists_kernels/_base.py

```diff
@@ -0,0 +1,198 @@
+# -*- coding: utf-8 -*-
+"""
+Abstract base class for pairwise transformers (such as distance/kernel matrix makers)
+"""
+
+__author__ = ["fkiraly"]
+
+
+from sktime.base import BaseEstimator
+
+from sktime.utils.validation.series import check_series
+
+
+class BaseTrafoPw(BaseEstimator):
```

Comment:
The word `Tabular` should be added to the end of the base class name, in my view.  (So either `BasePairwiseTransformerTabular` or `BasePairwiseTabular`).  Things are starting to get long and unwieldy, but I think its just about ok...

Reply:
hm, undecided here. @chrisholder, any strong opinions?

Reply:
further discussed in comment below